neuromation/api/images.py

```diff
@@ -103,6 +103,30 @@ async def push(
             raise  # pragma: no cover
         return remote
 
+    async def digest(self, remote: RemoteImage) -> str:
+        name = f"{remote.owner}/{remote.name}"
+        auth = await self._config._registry_auth()
+        assert remote.tag
+        url = self._registry_url / name / "manifests" / remote.tag
+        async with self._core.request(
+            "HEAD",
+            url,
+            auth=auth,
+            headers={"Accept": "application/vnd.docker.distribution.manifest.v2+json"},
+        ) as resp:
+            return resp.headers["Docker-Content-Digest"]
+
+    async def rm(self, remote: RemoteImage, digest: str) -> None:
+        try:
+            name = f"{remote.owner}/{remote.name}"
+            auth = await self._config._registry_auth()
+            url = self._registry_url / name / "manifests" / digest
+            async with self._core.request("DELETE", url, auth=auth) as resp:
+                assert resp
+        except DockerError as error:
```

Comment:
How is it possible to get DockerError from HTTP (aiohttp precisely) call?

Reply:
Good catch! That's a leftover from the very first draft of the code. Removed.